apps/prairielearn/src/pages/studentAssessment/studentAssessment.ts

```diff
@@ -36,6 +37,15 @@ router.use(logPageView('studentAssessment'));
 router.get(
   '/',
   asyncHandler(async function (req, res) {
+    if (!(res.locals.authz_result?.active ?? true)) {
+      // Student did not start the assessment, and the assessment is not active.
```

Comment:
It took me a while to figure out why you could assume that "student did not start the assessment", but it's because of the new redirect middleware. Worth a comment to clarify?

Reply:
Extremely fair, I tried to improve the comment in df2ae3b, is that clear now?

Reply:
That's clearer, thanks.